### DISCUSSION TITLE
Beta v8.4.9 released

### DISCUSSION THREAD

**Discussion by jcoffland** [FoldingAtHome/fah-client-bastet#308] [Announcements]
Changes:

## fah-web-client
   - Show ``Warning`` when more than one consecutive WU fails.

## fah-client
 - Recommend ``nvidia-opencl-icd`` package on Debian install. #293
 - Attempt graceful shutdown of client in Windows installer.  #290

**Comment by muziqaz** [comment]
alpha page still on 8.4.8
